src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts

```diff
@@ -185,6 +190,9 @@ class indexes extends viewModelBase {
     }
 
     private initObservables() {
+        this.localNodeTag(this.clusterManager.localNodeTag());
+        this.isCluster = ko.pureComputed(() => this.clusterManager.topology().nodes().length > 1);
```

Comment:
clusterTopologyManager has nodesCount, please use it.

Reply:
done